src/main/resources/application-dev.yaml

```diff
@@ -15,16 +15,24 @@ spring:
     enabled: true
   h2:
     console:
-      enabled: false
+      enabled: true
   jpa:
     properties:
       hibernate:
         format_sql: true
     show-sql: true
+  servlet:
+    multipart:
+      max-file-size: 128MB
+      max-request-size: 128MB
+      enabled: true
   mustache:
     check-template-location: false
   autoconfigure:
     exclude: org.keycloak.adapters.springboot.KeycloakAutoConfiguration
+docs:
+  storage:
+    directory: /Users/adityasampath/Downloads
```

Comment:
create a type also here. like `type: filesystem`

This one is used to save in the database and also for validation in the DocumentServiceImpl.

Reply:
don.e